sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/FunctionCall.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.openai.models;
+
+import com.azure.core.annotation.Immutable;
+
+@Immutable
+public class FunctionCall {
```

Comment:
Add javadoc for this class and also highlight the two constants that can be used (which are otherwise hard to discover).